axonserver/src/main/java/io/axoniq/axonserver/exception/PropagatingEventException.java

```diff
@@ -0,0 +1,21 @@
+/*
+ *  Copyright (c) 2017-2021 AxonIQ B.V. and/or licensed to AxonIQ B.V.
+ *  under one or more contributor license agreements.
+ *
+ *  Licensed under the AxonIQ Open Source License Agreement v1.0;
+ *  you may not use this file except in compliance with the license.
+ *
+ */
+
+package io.axoniq.axonserver.exception;
+
+/**
+ * @author Marc Gathier
+ * @since 4.4.13
+ */
+public class PropagatingEventException extends RuntimeException {
```

Comment:
I don't like the name of this exception because it doesn't express what happened but how this exception should be handled. Not sure what would be the best name for it, something in the line of `StopTheWorldException`, `CriticalInitializationException`, `IllegalOperationException`, `UnhealthyStateException`, ...